guides/getting-started.md

```diff
@@ -41,9 +41,35 @@ export class SomeModule { }
 
 Now, import the SCSS file from the package in `src/styles.scss`:
 
-```scss
-/* You can add global styles to this file, and also import other style files */
-
+``` scss
 @import "~@cashmere/sass/cashmere";
 ```
-:::
+
+#### Step 4. Add references to dependencies (optional)
+Add a reference to popper.js in `.angular-cli.json`
+``` json
+{
+    ...
+    "apps": [
+        {
+            ...
+            "scripts": [
+                "../node_modules/popper.js/dist/umd/popper.min.js"
+            ]
+            ...
+        }
+    ]
+    ...
+}
+```
+
+Add a reference to font awesome ([official instructions](https://github.com/angular/angular-cli/blob/master/docs/documentation/stories/include-font-awesome.md)) and open sans by adding a reference in `src/styles.scss`
+
+``` scss
+$fa-font-path: '../node_modules/font-awesome/fonts';
+@import '../node_modules/font-awesome/scss/font-awesome';
+
+$FontPathOpenSans: "../node_modules/npm-font-open-sans/fonts";
+@import '../node_modules/npm-font-open-sans/open-sans';
+```
+:::
```

Comment:
When I first set up to build Cashmere locally I had to do these additional steps for it to work -
Install angular-devkit/core
npm install d3 --save
Not sure if these are of value to other users or if this is even the right place for them but FYI.

Reply:
This needs to be added to the demo package.json, I will do that here

Reply:
Its not a requirement to use the library (as opposed to the demo app), so I think the getting started is fine the way it is